integration-tests/features/restart-container.feature

```diff
@@ -0,0 +1,11 @@
+Feature: container shold be running after restart
```

Comment:
I don't think this is really an independent feature, but rather a new scenario in the existing httpd-stateless migration tests.